### ISSUE TITLE
Problem when building UR with latest UMF

### ISSUE THREAD

**Issue by igchor** [oneapi-src/unified-memory-framework#480]
```
CMake Error at build/_deps/unified-memory-framework-src/CMakeLists.txt:21 (set_source_version):
 Unknown CMake command "set_source_version".
 ```
 
 Ref: https://github.com/oneapi-src/unified-runtime/actions/runs/9025858101/job/24802225188?pr=1430

**Comment by lukaszstolarczuk** [comment]
@igchor please verify your PR with my fix: https://github.com/oneapi-src/unified-memory-framework/pull/483

I've tested it locally and it passed.

**Comment by igchor** [comment]
> @igchor please verify your PR with my fix: #483
> 
> I've tested it locally and it passed.

It seems to work after the fix, thanks!
